### DISCUSSION TITLE
Welcome to paimon-webext Discussions!

### DISCUSSION THREAD

**Discussion by daidr** [daidr/paimon-webext#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by OtakuCat2020** [comment]
现在点击浏览器的派蒙小头像不进入验证码页面而是进入下面的页面，: (
(https://webstatic.mihoyo.com/app/community-game-records/rpg/?game_id=6&ref=paimon&mhy_presentation_style=fullscreen#/)#/

**Reply by daidr** [reply]
> 现在点击浏览器的派蒙小头像不进入验证码页面而是进入下面的页面，: (
> 
> (https://webstatic.mihoyo.com/app/community-game-records/rpg/?game_id=6&ref=paimon&mhy_presentation_style=fullscreen#/)#/

已修复，可以去chrome webstore下载0.2.1版本试试
